JS/Event-Listeners.js

```diff
@@ -23,4 +23,29 @@ function addEventListeners(readMoreBtn, movieDetails, stars) {
       };
     });
   }
+  //-------------------------
+  document.addEventListener("DOMContentLoaded", () => {
+    fetchMoviesFromAPI().then(fetchedMovies => {
```

Comment:
You can also use `await` instead of `then` here.

Reply:
If I use await instead of then , it throws an error. The error message is "await is not a function". Could you please tell me how I can use await ?

Reply:
Did you put `async` in the callback?

```js
document.addEventListener("DOMContentLoaded", async () => {
  const movies = await fetchMovies();
```